main.go

```diff
@@ -54,6 +56,44 @@ func sources() []string {
 }
 
 func main() {
+	var (
+		cfgFile        string
+		debug, version bool
+	)
+	flag.StringVar(&cfgFile, "c", "policies.yaml", "Path to configuration file")
```

Comment:
This is not necessarily a path to file. Can also be a local folder or Github URL (check the README to copy the sentence if you want)